src/app/body/all-products/product-card/product-card.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, Input, OnInit} from '@angular/core';
+// @ts-ignore
```

Comment:
The same with ts ignore. Please fix it in whole app.

Reply:
fixed